place topbars first in page-layout

tobibeer opened the pull request:
atm they are rendered last which is a problem for mobile / responsive ux

Jermolene commented:
@tobibeer we reorder the elements of the page template with the `list` field on `$:/tags/PageTemplate`. I'd be reluctant to start using two different conflicting techniques.

tobibeer commented:
> We reorder the elements of the page template with the list field on $:/tags/PageTemplate. I'd be reluctant to start using two different conflicting techniques.

I understand. I didn't even considered they could have been explicitly put last. Reverted the topbar tiddlers and modified the **list** field of the tag tiddler instead.

Jermolene commented:
Thanks @tobibeer
